src/DearImGui/Raw.hs

```diff
@@ -245,6 +253,48 @@ C.context (Cpp.cppCtx <> C.bsCtx <> imguiContext)
 C.include "imgui.h"
 Cpp.using "namespace ImGui"
 
+-- | Wraps @ImGuiListClipper
+newtype ImGuiListClipper = ImGuiListClipper (Ptr ())
```

Comment:
Add `ImGuiListClipper` to Structs.hs and wrap a pointer to it.

```haskell
newtype ListClipper = ListClipper (Ptr ImGuiListClipper)
```

This should help with `void*`'s and casts.

Reply:
Good idea!